src/components/InputForm/index.js

```diff
@@ -4,25 +4,27 @@ import classes from './style.module.css';
 
 class InputForm extends Component {
   state = {
-    content: ''
+    content: '',
   }
 
-  changeHandler = event => {
-    this.setState({content: event.target.value});
-  }
+  changeHandler = (event) => {
+    this.setState({ content: event.target.value });
+  };
 
   render() {
     const { content } = this.state;
     return (
-        <form className={classes.form}>
-          <label>Search for a medicine:</label>
+      <form className={classes.form}>
+        <label htmlFor="boom">Search for a medicine:
```

Comment:
It's generally not good practice to have text without a direct wrapping tag. Rather write as: 
```
<label htmlFor="boom">
  <span>Search for a medicine:</span>
  <input
     .... 
```